### PR TITLE
docs: document audience policy for forward-to endpoint

### PR DESCRIPTION
**Pull request by emmenko**
Documentation for #2667

### CODE REVIEW

File: website/src/content/api-reference/commercetools-frontend-application-shell.mdx [hunk 1]
```diff
@@ -194,6 +196,20 @@ const useExternalApiFetcher = () => {
 };
 ```
 
+Available options are:
```
Comment (CarlosCortizasCT) [inline website/src/content/api-reference/commercetools-frontend-application-shell.mdx:199]: Perhaps we could add a link to the new section that explains how to use this header (`/custom-applications/concepts/integrate-with-your-own-api#configuring-the-audience-policy`).

File: website/src/content/api-reference/commercetools-frontend-application-shell.mdx [hunk 2]
```diff
@@ -194,6 +196,20 @@ const useExternalApiFetcher = () => {
...
+  | 'forward-url-full-path'
+  | 'forward-url-origin';
+
+type TApolloContextProxyForwardTo = {
+  uri: string;
+  headers?: Record<string, string>;
+  audiencePolicy?: TForwardToAudiencePolicy;
+};
+```
+
```
Comment (emmenko) [inline website/src/content/api-reference/commercetools-frontend-application-shell.mdx:212]:
```suggestion

For more information see [configuring the audience policy](/concepts/integrate-with-your-own-api#configuring-the-audience-policy).

```

**Review by CarlosCortizasCT — APPROVED** [review]
Looks very good to me 👌

File: website/src/content/concepts/integrate-with-your-own-api.mdx [hunk 3]
```diff
@@ -256,7 +279,7 @@ The setup described above is not needed when using the `@commercetools-backend/e
 Developing a Custom Application locally, together with an external API, has some limitations.
 For example, using the `/proxy/forward-to` endpoint of the Merchant Center API Gateway with a target URL pointing to `http` is not allowed. Only `https` targets are allowed.
 
-The most straighforward solution is using the production URL of the external API, which may be inconvenient in the development processs.
+The most straightforward solution is using the production URL of the external API, which may be inconvenient in the development process.
```
Comment (kark) [inline website/src/content/concepts/integrate-with-your-own-api.mdx:282]: Thanks 🙏

**Pull request merged**